sql/core/src/test/resources/sql-tests/inputs/pipe-operators.sql

```diff
@@ -74,6 +74,10 @@ create temporary view windowTestData as select * from values
 -- SELECT operators: positive tests.
 ---------------------------------------
 
+-- Selecting from a constant.
+from t
+|> select 1 as x;
```

Comment:
The SQL grammar for the `fromClause` shows this:

```
fromClause
    : FROM relation (COMMA relation)* lateralView* pivotClause? unpivotClause?
    ;
```

Let's add a few test cases to cover the other optional components like `(COMMA relation)*` and `lateralView*` as well.